contrib/migration_script.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+"""
+This script is for migrating jobs to another namespace
+"""
+import argparse
+import subprocess
+import time
+
+import yaml
+from six.moves.urllib.parse import urljoin
+from six.moves.urllib.parse import urlparse
+
+from tron.commands import client
+
+
+class bcolors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Migrate jobs to new namespace'
+    )
+    parser.add_argument(
+        '--server',
+        required=True,
+        # default='http://tronplayground-uswest1adevc.dev.yelpcorp.com:8089',
+        help='specify the location of tron master'
+    )
+    parser.add_argument(
+        '--old-ns',
+        required=True,
+        # default='dsc-test-service',
+        help='Old namespace'
+    )
+    parser.add_argument(
+        '--new-ns',
+        required=True,
+        # default='test_namespace',
+        help='New namespace'
+    )
+    parser.add_argument(
+        'source',
```

Comment:
When would this ever not be `$old_ns.yaml`?

Reply:
The $source should have the full path, while $old_ns is only the name of the namespace. I was thinking the path can be 'yelpsoa-configs/namespace' or  'yelpsoa-configs/tron/norcal-prod', although I am not sure if we have use cases for the first one.